README.md

```diff
@@ -2,32 +2,39 @@
 
 [![Build Status](https://travis-ci.com/oscar-system/Polymake.jl.svg?branch=master)](https://travis-ci.com/oscar-system/Polymake.jl)
 
-Julia package for using [`polymake`](https://polymake.org/doku.php), a software for research in polyhedral geometry.
+`Polymake.jl` is a julia package for using [`polymake`](https://polymake.org/doku.php), a software for research in polyhedral geometry from julia.
 This package is developed as part of the [OSCAR](https://oscar.computeralgebra.de) project.
 
+The current version of `Polymake.jl` relies on `polymake` version `3.3` or later.
+
 **Index:**
 * [Installation](#installation)
 * [Examples](#examples)
 * [Polymake syntax translation](#polymake-syntax-translation)
-* [Current state](#current-state)
+* [Current state](#current-state-of-the-polymake-wrapper)
 
 ## Installation
 
-The installation can be done in the Julia REPL by first pressing `]` and then
+The installation can be done in the julia REPL by executing
 ```julia
-pkg> add Polymake
+julia> using Pkg; Pkg.add("Polymake")
 ```
 This will fetch a pre-build binary of `polymake`. You are ready to start `using Polymake`.
 
+Note: Pre-build binaries are available only for the `Linux` platform. MacOSX users need to follow the "Your own installation of `polymake`" below. Windows users are encouraged to try running `julia` inside Window Subsystem for Linux and reporting back ;)
+
 ### Your own installation of `polymake`
 
-If you already have a recent enough version of `polymake` on your system, you either need to make `polymake-config` available in your `PATH` or the environment variable `POLYMAKE_CONFIG` needs to point to the correct `polymake-config` file.
+If you already have a recent enough version of `polymake` on your system,
```

Comment:
Add the concrete polymake version here. 3.2? 3.3?

Reply:
`3.3`, but it's indicated in the third line of README, should I repeat this here?

Reply:
Cannot hurt